R/sensitivity_labels.R

```diff
@@ -1,9 +1,18 @@
+#' @keywords internal
+.get_sensitivity_xml_map <- function() {
```

Comment:
I don't think this is needed - sorry should have probably mentioned that James put in an internal dataset (`sysdata.rda`) that should contain the object `sensitivity_label_xml` so his code that used `sensitivity_label_xml[[label]]` should work to put in the xml code.